model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
The API hasn't really been broken; There is no promise that the file name returned from any specific function follows a specific format; Only that the name reflects the method used to generate that file, and I think that behaviour is still enforced. In other words, we are still naming the file after the method that ends up generating that file; Not the one that was originally requested.